### DISCUSSION TITLE
Now valid Migration guide to WalletKit gist

### DISCUSSION THREAD

**Discussion by JustGuitarKitty** [WalletConnect/walletconnect-monorepo#5411] [Developer Support]
Hi! Now link repeat AppKit migration guide :(

**Comment by rtomas** [comment]
Hi ! I can't understand. Can you share more information about your issue ?

**Reply by JustGuitarKitty** [reply]
Hi! Into doc has links to migration gist, WalletKit migration gist contains AppKit Migration gist link
![IMG_3596](https://github.com/user-attachments/assets/5f0abd35-7ef9-4b27-bc90-f9ac02bd0ceb)

**Reply by rtomas** [reply]
ok .. can you send me the link of that. thanks
